src/Command/NamespaceToPSR4Command.php

```diff
@@ -115,6 +115,10 @@ private function findFilesInPath(string $path): array
     private function resolveExpectedNamespace(string $namespaceRoot, SplFileInfo $fileInfo): string
     {
         $relativePathNamespace = str_replace('/', '\\', $fileInfo->getRelativePath());
+        if ($relativePathNamespace === '') {
```

Comment:
This fixes the issue where the namespace of `src\Kernel.php` gets changed from `App` to `App\`